handout/testing/services/mockbackend.md

```diff
@@ -0,0 +1,115 @@
+# Testing HTTP Requests Using MockBackend
+
```

Comment:
This seems a little wordy, wondering if something like

> To unit test our services, we don't want to make actual HTTP requests. To accomplish this, we need to mock out our HTTP services. Angular 2 provides us with a `MockBackend` class that can be configured to provide mock responses to our requests, without actually making a network request. 
>
> The configured `MockBackend` can then be injected into HTTP, so any calls to the service, such as `http.get` will return our expected data, allowing us to test our service in isolation from real network traffic.

is any better? ... might need some tweaks still - just feels like the last part of the first paragraph is largely repeating the first part of the second.